compiler/src/iree/compiler/Pipelines/Options.cpp

```diff
@@ -112,8 +112,12 @@ InputDialectOptions::Type InputDialectOptions::parseInputTypeMnemonic() {
 
 void PreprocessingOptions::bindOptions(OptionsBinder &binder) {
   static llvm::cl::OptionCategory category(
+      "Preprocessing options",
       "IREE options for apply custom preprocessing before normal IREE "
-      "compilation flow");
+      "compilation flow. These options are exercised in the following order:"
+      " 1. `iree-preprocessing-pass-pipeline`,"
+      " 2. `iree-preprocessing-transform-spec-filename`,"
+      " 3. `iree-preprocessing-pdl-spec-filename`");
```

Comment:
Does this get formatted with newlines between each list item:
```
1. `foo`
2. `bar`
3. `baz`
```

or like ``1. `foo` 2. `bar` 3. `baz` ``?

Might need to insert `\n` characters

Reply:
Done.